boards/avr-rss2/Makefile.dep

```diff
@@ -0,0 +1,6 @@
+USEMODULE += boards_common_atmega
+
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += bmx280
```

Comment:
```suggestion
  USEMODULE += bme280
```

`bmx280` is the generic driver, depending on whether you use `USEMODULE += bme280` or `USEMODULE += bmp280` different code paths are activated by `#ifdef`s in the driver.

Reply:
Thanks. OK I see now. The driver is also checking chip-id. So it's a "feature" of the driver allow inconsistency. Yes patch needed. Anything more?

Reply:
Actually we should probably add a warning if a pseudomodule is used directly…